cli/cmd/install_test.go

```diff
@@ -180,31 +180,40 @@ func TestRender(t *testing.T) {
 	}
 	withProxyIgnoresOptions.ignoreInboundPorts = []string{"22", "8100-8102"}
 	withProxyIgnoresOptions.ignoreOutboundPorts = []string{"5432"}
-	withProxyIgnoresValues, _, _ := withProxyIgnoresOptions.validateAndBuild("", nil)
+	withProxyIgnoresValues, _, _ := withProxyIgnoresOptions.validateAndBuild("", withProxyIgnoresOptions.recordableFlagSet())
 	addFakeTLSSecrets(withProxyIgnoresValues)
 
 	withHeartBeatDisabled, err := testInstallOptions()
 	if err != nil {
 		t.Fatalf("Unexpected error: %v\n", err)
 	}
 	withHeartBeatDisabled.disableHeartbeat = true
-	withHeartBeatDisabledValues, _, _ := withHeartBeatDisabled.validateAndBuild("", nil)
+	withHeartBeatDisabledValues, _, _ := withHeartBeatDisabled.validateAndBuild("", withHeartBeatDisabled.recordableFlagSet())
 	addFakeTLSSecrets(withHeartBeatDisabledValues)
 
+	// FIXME: test heartbeat override here
```

Comment:
Should we fix this or remove the comment?

Reply:
remove the comment :)